repair/repair_task.hh

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include "repair/repair.hh"
+#include "tasks/task_manager.hh"
+
+class repair_task_impl : public tasks::task_manager::task::impl {
+public:
+    repair_task_impl(tasks::task_manager::module_ptr module, tasks::task_id id, unsigned sequence_number = 0, std::string keyspace = "", std::string table = "", std::string type = "", std::string entity = "", tasks::task_id parent_id = tasks::task_id::create_null_id())
```

Comment:
Why so many default parameters. I noticed this being a repeating pattern in task related code. Is there some user of this API which is significantly helped by all this?